2 - ListsLogicAndIterations/Code/Dictionarys.cs

```diff
@@ -3,21 +3,26 @@
 
 namespace ListsLogicAndIterations.Code
 {
-	public static class Dictionarys
-	{
-		public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
-		{
-			throw new NotImplementedException();
-		}
+    public static class Dictionarys
+    {
+        public static Dictionary<Guid, decimal> Build(Guid key, decimal value)
+        {
+            Dictionary<Guid, decimal> dict = new Dictionary<Guid,decimal>();
+            dict.Add(key, value);
+            return dict;
```

Comment:
You can short hand this into `return new Dictionary<Guid, decimal> { {key, value} };` There is nothing wrong with what you did, but the code in my example compiles into what you did and is a bit easier to maintain.